1.0.0 doc changes

cs-abhishek-shukla opened the pull request:
- Fixed Typo, Grammar, label names, doc structure, and information flow issues

cs-suyog-jinturkar commented on README.md:
```diff
@@ -3,13 +3,15 @@
 # Overview
 
-The `SOC Experience` Solution Pack is to simplify the process of setting up a demo environment. Instead of manually configuring various components and settings, this solution pack initiates the setup process also provides set of playbooks that to automatically trigger scenarios for the deployed solution packs.
+The **SOC Experience** solution pack comes with pre-configured playbooks that automatically trigger scenarios in deployed solution packs. Instead of manually configuring various components and settings, this solution pack initiates the setup process and comes with a set of playbooks that automatically trigger scenarios for already deployed solution packs. 
```
Overview should also cover the few SPs are also getting deployed as part of this SP and more info around it.

cs-rohit-khune commented on docs/setup.md:
```diff
@@ -11,26 +11,29 @@
-| SOC Overview Metrics | Required for SOC Management Widget and Dashboard |
-| Threat Intel Management | Required for Threat Intellgence Modules and Reports |
-| Continuous Delivery | Required to demostrate CICD |
+| Solution Pack Name                       | Version | Purpose                                                                          |
```
@cs-abhishek-shukla As discussed, refer to the attached 
[SP Name.xls](https://github.com/fortinet-fortisoar/solution-pack-soc-experience/files/11850829/SP.Name.xls)
SP versions

cs-rohit-khune commented on README.md:
```diff
@@ -3,13 +3,15 @@
+The **SOC Experience** solution pack comes with pre-configured playbooks that automatically trigger scenarios in deployed solution packs. Instead of manually configuring various components and settings, this solution pack initiates the setup process and comes with a set of playbooks that automatically trigger scenarios for already deployed solution packs. 
 
-The solution pack also configures data ingestion for the MITRE ATT&CK. this ingestion creates up to date records of MITRE related modules such as Techniques, Sub-Techniques, Tactics, Groups and Softwares
+Accelerate your learning curve, test real-world incidents, and maximize efficiency with predefined setups. Customize your testing environment, stay ahead of emerging threats, and future-proof your security operations. Simplify the process, save time, and unleash the true potential of FortiSOAR&trade;. Elevate your security operations with ease, effectiveness, and peace of mind. Experience FortiSO ...
```
I believe it may not fully convey the intended message. This SP only sets up a demo instance. @cs-abhishek-shukla Can we please rephrase or simplify this?

cs-abhishek-shukla committed:
1.0.0 doc changes
- Fixed Typo, Grammar, label names, doc structure, and information flow issues
- Incorporated PR review comments
